src/NHibernate.Test/PropertyRef/ManyToManyPropertyRefFixture.cs

```diff
@@ -131,7 +131,7 @@ bei System.ThrowHelper.ThrowKeyNotFoundException()
    bei NHibernate.Type.EntityType.LoadByUniqueKey(String entityName, String uniqueKeyPropertyName, Object key, ISessionImplementor session) in C:\Users\Armin\Projects\NHibernate\branches\2.1.x\nhibernate\src\NHibernate\Type\EntityType.cs:Zeile 552.
 			 */
 
-			Assert.That(loadedManyA.ManyBs.Count, Is.EqualTo(3));
+			Assert.That(loadedManyA.ManyBs, Has.Count.EqualTo(3).And.None.Null);
```

Comment:
`And.None.Null` is required because there was two bugs: one causing an exception (the reported one) and another one, prevented by the failure, causing the children to be null with this test case.